cmd/exo/cmd/privnet_create.go

```diff
@@ -102,15 +128,18 @@ func privnetCreate(name, desc, zoneName string) error {
 	newNet := resp.(*egoscale.Network)
 
 	table := table.NewTable(os.Stdout)
-	table.SetHeader([]string{"Name", "Description", "ID"})
-	table.Append([]string{newNet.Name, newNet.DisplayText, newNet.ID.String()})
+	table.SetHeader([]string{"Name", "Description", "ID", "StartIP", "EndIP", "Netmask"})
+	table.Append([]string{newNet.Name, newNet.DisplayText, newNet.ID.String(), newNet.StartIP.String(), newNet.EndIP.String(), newNet.Netmask.String()})
```

Comment:
`<nil>` is an implementation details that shouldn't be shown.

| NAME | DESCRIPTION |                  ID                  | STARTIP | ENDIP | NETMASK |
|------|-------------|--------------------------------------|---------|-------|---------|
| bar  |             | d8283a3a-a9ff-4f5d-b385-f7ad16d4c311 | \<nil>   | \<nil> | \<nil>   |